src/ORM/Association/HasMany.php

```diff
@@ -182,6 +182,174 @@ public function saveAssociated(EntityInterface $entity, array $options = [])
     }
 
     /**
+     * Associates the source entity to each of the target entities provided.
+     * When using this method, all entities in `$targetEntities` will be appended to
+     * the source entity's property corresponding to this association object.
+     *
+     * This method does not check link uniqueness.
+     *
+     * ### Example:
+     *
+     * ```
+     * $user = $users->get(1);
+     * $allArticles = $articles->find('all')->execute();
+     * $users->Articles->link($user, $allArticles);
+     * ```
+     *
+     * `$user->get('articles')` will contain all articles in `$allArticles` after linking
```

Comment:
Well, it would not be that difficult to guess it when you look at the return type and realize that it returns bool, but, once explicit is better than implicit,  I can make it clearer at the explanation.